scripts/content-auditor/README.md

```diff
@@ -0,0 +1,38 @@
+# Content Auditor
+
+## Description
+
+The Tina documentation is exhaustive and contains mentions of planned future features.
+To check these at once we've created this script that uses every document or blog file as content to be run against a LLM prompt.
```

Comment:
Make this more generic

```suggestion
The Tina documentation is exhaustive. To make maintenance and spotting issues easy, we've created this script that runs a LLM prompt against every .mdx file it can find.
```